demisto_sdk/commands/lint/resources/pylint_plugins/base_checker.py

```diff
@@ -449,6 +449,47 @@ def _check_if(comp_with):
         except Exception:
             pass
 
+    # -------------------------------------------- Match Node ---------------------------------------------
+
+    def _commands_in_match_statement_checker(self, node):
+        """
+        Args: node which is a Match Node.
+        Check all possible appearances of implementations of commands in an Match case statement:
+        - if command exists in a regular match case e.g. >>> case 'command1':
+        - if command exists in a match case with an or operator e.g. >>> case 'command1' | 'command2':
+
+        Adds the relevant error message using `add_message` function if none of the above exists.
+        """
+
+        def _check_match(comp_with):
+            """
+            Internal function that inferences the value of the comp_with argument.
+            If the inferred value is a command which is in the commands list, removes it , as we found an implementation
+            """
+            # for regular if 'command' == command with inference mechanize
+            commands = self._infer_name(comp_with)
+
+            for command in commands:
+                if command in self.commands:
+                    self.commands.remove(command)
+
+                if not self.test_module_implemented and command == TEST_MODULE:
+                    self.test_module_implemented = True
+
+        try:
+            for case in node.cases:
+                # running on astroid.Match
+                # for "case 'command1' | 'commands2'"
+                if isinstance(case.pattern, astroid.MatchOr):
+                    for match_value in case.pattern.patterns:
+                        _check_match(match_value.value)
+
+                elif isinstance(case.pattern, astroid.MatchValue):
+                    _check_match(case.pattern.value)
+
+        except Exception:
+            pass
```

Comment:
```suggestion
        except Exception as e:
            logger.warning(f"....")
```
log the exception / error here